openedx/features/course_experience/course_tools.py

```diff
@@ -57,6 +57,23 @@ def url(cls, course_key):
         """
         raise NotImplementedError("Must specify a url for a course tool.")
 
+    @classmethod
+    def http_method(cls):
```

Comment:
NIT: could we instead use a property, maybe in the init, like self.requires_form = False.  may be more understandable?  idk just my preference

Reply:
Updated